content/collections/flutter_sdk/en/flutter-sdk-4-0.md

```diff
@@ -35,6 +35,17 @@ Add `platform :ios, '13.0'` to your Podfile. Run `pod install` under the ios d
 
 To enable Bitcode, follow Flutter's [documentation](https://github.com/flutter/flutter/wiki/Creating-an-iOS-Bitcode-enabled-app).
 
+### macOS installation
+
+Add `platform :osx, '10.15'` to your Podfile. Run `pod install` under the macos directory of your Flutter project to update the CocoaPods dependencies.
```

Comment:
```suggestion
Add `platform :osx, '10.15'` to your Podfile. Run `pod install` in the macOS directory of your Flutter project to update the CocoaPods dependencies.
```

Reply:
`macos` is not capitalized in the generated folder by flutter so I'll leave it uncapitalized. I'll change this line and the above to use `in` rather than `under`